scripts/sequence_labeling/finetune_bert.py

```diff
@@ -77,6 +77,8 @@ def parse_args():
                             help='Learning rate for optimization')
     arg_parser.add_argument('--warmup-ratio', type=float, default=0.1,
                             help='Warmup ratio for learning rate scheduling')
+    arg_parser.add_argument('--tagging-first-token', type=str2bool, default=True,
```

Comment:
How about `parser.add_argument('--tag-last-token', action='store_true')`. It seems simpler to call `finetune_bert.py --tag-last-token` than `finetune_bert.py --tagging-first-token=False`.

In either case please update the test case in `scripts/tests/` to run invoke the `finetune_bert.py` with both options. You can parametrize the test following for example Haibin's recent PR: https://github.com/dmlc/gluon-nlp/pull/1121/files#diff-fa82d34d543ff657c2fe09553bd0fa34R234

Reply:
Sure, I will update it.